x-pack/legacy/plugins/epm/server/plugin.ts

```diff
@@ -16,7 +16,9 @@ import { routes } from './routes';
 
 export { createSetupShim } from './shim';
 
-export type EPMPluginInitializerContext = Pick<PluginInitializerContext, 'config'>;
+export interface EPMPluginInitializerContext {
+  config: Omit<PluginInitializerContext['config'], 'legacy'>;
```

Comment:
This is totally valid, but do you mind changing it to only pull in the items we're using?

```suggestion
  config: Pick<PluginInitializerContext['config'], 'create' | 'createIfExists'>;
```

It's supposed to be temporary so I'm trying to keep it as small as possible